container-registry/aws/ecr/versions.tf

```diff
@@ -1,13 +1,24 @@
+provider "skopeo2" {
+  destination {
+    login_username = data.aws_ecr_authorization_token.current.user_name
+    login_password = data.aws_ecr_authorization_token.current.password
+  }
+}
+
 terraform {
   required_version = ">= 1.0"
   required_providers {
     aws = {
       source  = "hashicorp/aws"
       version = ">= 5.4.0"
     }
-    null = {
-      source  = "hashicorp/null"
-      version = ">= 3.2.1"
+    skopeo2 = {
+      source  = "bsquare-corp/skopeo2"
+      version = ">= 1.1.1"
+    }
+    generic = {
+      source  = "aneoconsulting.github.io/aneoconsulting/generic"
```

Comment:
You did not use this provider

Reply:
here https://github.com/aneoconsulting/ArmoniK.Infra/blob/5cbe4d69819f71713b20974ea21f09a39dd67b6f/container-registry/aws/ecr/main.tf#L137